vtex-sites/base.store
# fix: CSS Warnings

## What's the purpose of this pull request?
This PR is part in a series of PRs aiming to improve Developer Perception about base.store. This PR focusses on removing those annoying CSS Warnings and solves possible issues with import order and class crashes.

## How does it work?
The warnings were being thrown because the way we were importing CSS files. Importing all css files at once in a single `components.scss` file solves the issue. Also, this makes it work on next.js and other frameworks.

If you are adding a new component, please import the component's css into the `components.scss` instead on the component itself. 

If you are wondering if performance will be harmed in any ways, the answer is no, because we were already using a single CSS anyways.

Other solutions using CSS modules may be attempted in the future for better css splitting if it's needed down the line

## How to test it?
Make sure all styles appear on the store. Also, make sure the warnings went away.


## Checklist
